src/commands/Fun/playedmyself.ts

```diff
@@ -0,0 +1,21 @@
+import { SteveCommand } from '@lib/structures/commands/SteveCommand';
+import { CommandStore, KlasaMessage } from 'klasa';
+import { Message } from 'discord.js';
+
+export default class extends SteveCommand {
+
+	public constructor(store: CommandStore, file: string[], directory: string) {
+		super(store, file, directory, {
+			cooldown: 60,
+			cooldownLevel: 'channel',
+			aliases: ['i played myself'],
+			description: 'congratulations, you played yourself.',
+			extendedHelp: 'this command returns the DJ Khaled GIF'
```

Comment:
few changes needed here: command options should be alphabetized, description + extendedHelp should be capitalized properly, and the alias shouldn't have any spaces